e2e/tests/runtime/runtime-status-update.spec.ts

```diff
@@ -0,0 +1,222 @@
+import HelpSidebar from 'app/component/help-sidebar';
+import RuntimePanel, {
+  ComputeType,
+  RuntimePreset,
+  StartStopIconState
+} from 'app/component/runtime-panel';
+import {config} from 'resources/workbench-config';
+import {createWorkspace, signIn} from 'utils/test-utils';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import {makeRandomName} from 'utils/str-utils';
+import NotebookPreviewPage from 'app/page/notebook-preview-page';
+
+// This one is going to take a long time.
+jest.setTimeout(60 * 30 * 1000);
+
+describe('Updating runtime parameters', () => {
+  beforeEach(async () => {
+    await signIn(page);
+    const workspaceCard = await createWorkspace(page, config.altCdrVersionName);
```

Comment:
Unfortunately have to do this each time because there's no good way to tell from a workspace card what CDR the workspace is using and we need the one with microarray data